ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 2018.3.9f1
+m_EditorVersion: 2018.3.11f1
```

Comment:
Can we keep the Unity version to .9f1 please, for now?

Reply:
I pushed the fix to the memory leak problem. 
Yes I can put the version at .9f1

Reply:
version changed back to 9f1